docs/reference/analysis/tokenfilters/condition-tokenfilter.asciidoc

```diff
@@ -1,88 +1,144 @@
 --------------------------------------------------
-// TEST[continued]
-
-And it'd respond:
 
+/////////////////////
 [source,console-result]
 --------------------------------------------------
 {
-  "tokens": [
+  "tokens" : [
+    {
+      "token" : "the",
+      "start_offset" : 0,
+      "end_offset" : 3,
+      "type" : "<ALPHANUM>",
+      "position" : 0
+    },
     {
-      "token": "what",              <1>
-      "start_offset": 0,
-      "end_offset": 4,
-      "type": "<ALPHANUM>",
-      "position": 0
+      "token" : "QUICK",
+      "start_offset" : 4,
+      "end_offset" : 9,
+      "type" : "<ALPHANUM>",
+      "position" : 1
     },
     {
-      "token": "Flapdoodle",        <2>
-      "start_offset": 5,
-      "end_offset": 15,
-      "type": "<ALPHANUM>",
-      "position": 1
+      "token" : "BROWN",
+      "start_offset" : 10,
+      "end_offset" : 15,
+      "type" : "<ALPHANUM>",
+      "position" : 2
+    },
+    {
+      "token" : "fox",
+      "start_offset" : 16,
+      "end_offset" : 19,
+      "type" : "<ALPHANUM>",
+      "position" : 3
     }
   ]
 }
 --------------------------------------------------
+/////////////////////
+
+[[analysis-condition-tokenfilter-configure-parms]]
+==== Configurable parameters
+
+`filter`::
++
+--
+(Required, array of token filters)
+Array of token filters. If a token matches the predicate script in the `script`
+parameter, these filters are applied to the token in the order provided.
 
-<1> The term `What` has been lowercased, because it is only 4 characters long
-<2> The term `Flapdoodle` has been left in its original case, because it doesn't pass
-    the predicate
+These filters can include custom token filters defined in the index mapping.
+--
+
+`script`::
++
+--
+(Required, <<modules-scripting-using,script object>>)
+Predicate script used to apply token filters. If a token
+matches this script, the filters in the `filter` parameter are applied to the
+token.
+
+Only inline scripts are supported. For valid parameters, see
+<<_script_parameters>>.
+--
+
+[[analysis-condition-tokenfilter-customize]]
+==== Customize and add to an analyzer
+
+To customize the `condition` filter, duplicate it to create the basis
+for a new custom token filter. You can modify the filter using its configurable
+parameters.
+
+For example, the following <<indices-create-index,create index API>> request
+uses a custom `condition` filter to configure a new
+<<analysis-custom-analyzer,custom analyzer>>.
+
+[source,console]
+--------------------------------------------------
+PUT /palindrome_list
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "whitespace_reverse_first_token": {
+          "tokenizer": "whitespace",
+          "filter": [ "reverse_first_token" ]
+        }
+      },
+      "filter": {
+        "reverse_first_token": {
+          "type": "condition",
+          "filter": [ "reverse" ],
+          "script": {
+            "source": "token.getPosition() === 0"
```

Comment:
Maybe add a short comment after this example explaining what this analyzer does, its pretty obvious from the analyzer name but for some folks it might be worth spelling it out.

Reply:
Thank you for the comment! I was debating whether that was needed.

I added a sentence to the preceding paragraph with 3653233c2e1.